augly/tests/image_tests/functional_unit_test.py

```diff
@@ -17,7 +17,7 @@ def test_blur(self):
 
     def test_brightness(self):
         self.evaluate_function(imaugs.brightness)
-
+        
```

Comment:
remove indent

Reply:
@tanujdhiman you still need to remove the indent.